blocks/product-gallery-block/features/product-gallery/default.jsx

```diff
@@ -43,63 +47,75 @@ export function ProductGalleryDisplay({
 	});
 
 	return (
-		<Carousel
-			className={BLOCK_CLASS_NAME}
-			goToSlidePhrase={
-				/* istanbul ignore next */ (slideNumber) =>
-					phrases.t("product-gallery.go-to-slide", { slideNumber })
-			}
-			id={id}
-			indicators={indicatorType}
-			key={id}
-			label={phrases.t("product-gallery.aria-label")}
-			thumbnails={thumbnailsArray}
-		>
-			{shortenedCarouselItems?.map((item, carouselIndex) => {
-				const { _id: itemId, auth, alt_text: altText } = item;
-
-				// is this a featured image?
-				const isFeaturedImage = isFeaturedImageEnabled && carouselIndex === 0;
-
-				// take in app version that's the public key for the auth object in resizer v2
-				const targetAuth = auth[resizerAppVersion];
-				return (
-					<Carousel.Item
-						key={item.testId || itemId}
-						className={isFeaturedImage ? `${BLOCK_CLASS_NAME}__featured-slide` : ""}
-						label={`${phrases.t("product-gallery.slide-indicator", {
-							current: carouselIndex + 1,
-							maximum: carouselItems.length,
-						})}`}
-					>
-						<Image
-							alt={altText}
-							src={imageANSToImageSrc(item)}
-							resizedOptions={{ auth: targetAuth }}
-							width={375}
-							height={375}
-							responsiveImages={[150, 375, 500, 1500, 2000]}
-							resizerURL={resizerURL}
-							sizes={[
-								{
-									// featured image should take up full width of the screen on mobile and desktop
-									isDefault: true,
-									sourceSizeValue: "100vw",
-								},
-								...(isFeaturedImage
-									? []
-									: [
-											{
-												sourceSizeValue: "50vw",
-												mediaCondition: "(min-width: 48rem)",
-											},
-									  ]),
-							]}
-						/>
-					</Carousel.Item>
-				);
-			})}
-		</Carousel>
+		<div>
```

Comment:
This can be a fragment to reduce DOM elements.

Reply:
Good call! Changed to a fragment.